docs/src/Decorators/BooleanField.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
The decorators are fairly trivial - not sure we need a different menu for each.  We could put in one page and then use the right menu for navigation.